WeatherTests/UnitTests/CityValidatorTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherBL.Validators;
+
+namespace WeatherTests.UnitTests
+{
+    [TestFixture]
+    public class CityValidatorTests
+    {
+        private CityValidator cityValidator;
+        private string city;
+        private string nullCity;
+
+        public CityValidatorTests()
+        {
+            cityValidator = new CityValidator();
+            city = "London";
+            nullCity = null;
+        }
+
+        [Test]
+        public void Validate_CityNameIsNotNull_ReturnsTrue()
```

Comment:
I can see returns rtue in name but in assert IsNotNull

Reply:
Renamed